dopex-io/elvarg
# feat: merkl integration (dop-525)

Ease the process of claiming rewards from Merkl by adding the claim button on our dapp. We also display the reward APR.

## Scope

[dop-525](https://linear.app/dopex/issue/DOP-525/add-merkl-rewards-to-our-dapp)

## Implementation

Use Merkl's API to get transaction data including merkl proof and reward tokens associated with an address, chain ID and selected options pool's uni v3 pool.

## Screenshots

|         | before | after |
| ------- | ------ | ----- |
| desktop |  <img width="1728" alt="Screenshot 2024-01-08 at 6 48 06 PM" src="https://github.com/dopex-io/elvarg/assets/85767768/4cd1c888-0c48-4705-9c6b-8ba5e80a2d9f">  |   <img width="1728" alt="Screenshot 2024-01-08" src="https://github.com/dopex-io/elvarg/assets/85767768/b6208006-fc72-4d3d-b837-2f366ca046b0"> |
| mobile  |        |       |

## How to Test

- Visit app.dopex.io/clamm
- Verify Reward APRs
- Test Claim button
